app/views/dashboard/_compliance_host_reports_widget.html.erb

```diff
@@ -1,5 +1,5 @@
 <h4 class="ca"><%= _('Latest compliance reports') %></h4>
-<% latest_reports = ForemanOpenscap::ArfReport.latest %>
+<% latest_reports = ForemanOpenscap::ArfReport.latest.limit(9) %>
```

Comment:
It is the dashboard's [favourite number](https://github.com/theforeman/foreman/blob/develop/app/services/dashboard/data.rb#L19) :-)